### PR TITLE
[7.1.0] Add note about mock service sub context

### PR DESCRIPTION
**Pull request by arunans23**
Fixes https://github.com/wso2/docs-ei/issues/1854

### CODE REVIEW

**Review by Nirothipan — APPROVED** [review]

**Review by sajithaliyanage — APPROVED** [review]

File: en/micro-integrator/docs/develop/creating-unit-test-suite.md [hunk 1]
```diff
@@ -172,6 +172,18 @@ Mock services give the opportunity to simulate the actual endpoint.
...
   
+    !!! Note 
+        
+        Please note that the mock service **should have a sub context** with '/' defined, and additional sub contexts should be defined after that.
+        
+        - If you are trying to mock an endpoint `http://petstore.com/pets`, the wizard should look like below now.
+        
+        ![Mock Service with one sub context](../assets/img/create_project/synapse_unit_test/mockservice-context-sample.PNG) 
+        
+        - If you you are going to mock the same endpoint with an additional sub context (Eg : `http://petstore.com/pets/id` ), you can add it to the same mock service like below.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/develop/creating-unit-test-suite.md:183]:
```suggestion
        - If you are going to mock the same endpoint with an additional sub context (e.g., `http://petstore.com/pets/id` ), you can add it to the same mock service as shown below. 
```

File: en/micro-integrator/docs/develop/creating-unit-test-suite.md [hunk 2]
```diff
@@ -172,6 +172,18 @@ Mock services give the opportunity to simulate the actual endpoint.
     -   **Header Value**: Response request header value.
     -   **Send Out Response Payload**: Expected response payload from the service.   
   
+    !!! Note 
+        
+        Please note that the mock service **should have a sub context** with '/' defined, and additional sub contexts should be defined after that.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/develop/creating-unit-test-suite.md:177]: Shall we explain why we should do this?

Comment (arunans23) [reply en/micro-integrator/docs/develop/creating-unit-test-suite.md:177]: According to current implementation, this is how it works. There is no standard to define like this. Should we point the technical limitation here?

Comment (samgnaniah) [reply en/micro-integrator/docs/develop/creating-unit-test-suite.md:177]: No we should avoid the technical limitation. I assumed there might be a positive side of defining it like this.

**Review by samgnaniah — APPROVED** [review]

**Comment by arunans23** [comment]
Add PRs for other branches: https://github.com/wso2/docs-ei/pull/2581, https://github.com/wso2/docs-ei/pull/2583

**Pull request merged**